packages/calcite-components/src/components/dialog/dialog.tsx

```diff
@@ -270,6 +273,16 @@ export class Dialog
     updateFocusTrapElements(this);
   }
 
+  /**
+   * When defined, provides a condition for focusTrapDisabled override.
+   *
+   * @private
+   */
+  @method()
```

Comment:
@Elijbet this shouldn't use `@method`, it should be like the `onOpen` in the `openCloseComponent` interface.